internal/service/advancedcluster/resource_advanced_cluster_migration_test.go

```diff
@@ -72,7 +72,7 @@ func TestMigAdvancedCluster_geoShardedOldSchemaUpdate(t *testing.T) {
 	)
 
 	resource.ParallelTest(t, resource.TestCase{
-		PreCheck:     func() { mig.PreCheckBasic(t) },
+		PreCheck:     acc.SkipIfTPFAdvancedCluster(t, func() { mig.PreCheckBasic(t) }),
```

Comment:
do we need to enclose it with "func)?  not done in the previous change

Reply:
changed in 053deb331cb7c3189a7de3ffa47e597ba5721ac1.
It was done since the signature was expecting func()